diff --git a/app/models/user.rb b/app/models/user.rb
index 3a274f0..3b855d5 100644
--- a/app/models/user.rb
+++ b/app/models/user.rb
@@ -36,7 +36,7 @@ class User < ActiveRecord::Base
 
   validates :username, presence: true, if: :username_required?
   validates :username, uniqueness: { scope: :registering_with_oauth }, if: :username_required?
-  validates :document_number, uniqueness: { scope: :document_type }, allow_nil: true
+  validates :document_number, uniqueness: { scope: :document_type }, allow_blank: true
 
   validate :validate_username_length
 
